guide/payments/request.md

```diff
@@ -0,0 +1,477 @@
+   layout = "shadow"
+   caption = "Home screens can speed up the generating of a single-use payment request."
+   alt-text = "Screen showing home screen number pad entry."
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/payments/request/single-use-payment-request.png"
+   retina = "/assets/images/guide/payments/request/single-use-payment-request@2x.png"
+   layout = "shadow"
+   caption = "A newly generated single-use payment request."
+   alt-text = "Screen showing single-use payment request."
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/payments/request/single-use-payment-request-settings.png"
+   retina = "/assets/images/guide/payments/request/single-use-payment-request-settings@2x.png"
+   layout = "shadow"
+   caption = "Let users configure the settings of single-use payment requests."
+   alt-text = "Screen showing single-use payment request settings."
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/payments/request/single-use-transaction-request.png"
+   retina = "/assets/images/guide/payments/request/single-use-transaction-request@2x.png"
+   layout = "shadow"
+   caption = "A newly generated single-use transaction-request."
+   alt-text = "Screen showing newly generated single-use transaction-request."
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/payments/request/single-use-transaction-request-settings.png"
+   retina = "/assets/images/guide/payments/request/single-use-transaction-request-settings@2x.png"
+   layout = "shadow"
+   caption = "Let users configure the settings of the single-use transaction-request."
+   alt-text = "Screen showing newly generated single-use transaction-request settings."
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/payments/request/single-use-address-type.png"
+   retina = "/assets/images/guide/payments/request/single-use-address-type@2x.png"
+   layout = "shadow"
+   caption = "Supporting every address type benefits users and makes your wallet more interoperable."
+   alt-text = "Screen showing different address types."
+   width = 250
+   height = 541
+%}
+
+</div>
+
+## Reusable payment requests
+
+Reusable payment requests aren’t as widely adopted or used as their single-use counterparts. For some use cases, reusable payment requests offer an improved user experience, more flexibility, and enable unique use cases compared to single-use payment requests. Reusable payment requests avoids users from having to manually initiate a payment request each time they need to receive bitcoin, greatly simplifying payment flows.
+
+Reusable payment requests are common when making payments as they are made more frequently. So manually generating a payment request for each payment adds a lot of friction.
```

Comment:
This sentence only makes sense if the user thinks of payments as frequent, low-amount transfers. However, this is not defined anywhere. Goes back the to payments/transactions discussion.